Libplanet/Net/Swarm.cs

```diff
@@ -613,12 +626,17 @@ void ISwarm.ReplyMessage(Message message)
         /// </returns>
         /// <remarks>This does not render downloaded <see cref="IAction"/>s, but fills states only.
         /// If you want to render all <see cref="IAction"/>s from the genesis block to the recent
-        /// blocks use <see cref="StartAsync(TimeSpan, TimeSpan, CancellationToken)"/> method
-        /// instead.</remarks>
+        /// blocks use
+        /// <see cref="StartAsync(TimeSpan, TimeSpan, EventHandler{PreloadBlockDownloadFailEventArgs}, CancellationToken)"/>
+        /// method instead.</remarks>
+        /// <exception cref="AggregateException">Thrown when the given the block downloading is
+        /// failed and if <paramref name="preloadBlockDownloadFailed "/> is <c>null</c>.</exception>
+#pragma warning restore MEN002 // Line is too long
         public Task PreloadAsync(
             TimeSpan? dialTimeout = null,
             IProgress<PreloadState> progress = null,
             IImmutableSet<Address> trustedStateValidators = null,
+            EventHandler<PreloadBlockDownloadFailEventArgs> preloadBlockDownloadFailed = null,
```

Comment:
As this method in itself is named `PreloadAsync`, the prefix `preload` in this parameter seems redundant.

```suggestion
            EventHandler<PreloadBlockDownloadFailEventArgs> blockDownloadFailed = null,
```

Reply:
Fixed! 1a7555f